lib/jxl/image_metadata.h

```diff
@@ -377,6 +377,34 @@ struct CodecMetadata {
 
   size_t xsize() const { return size.xsize(); }
   size_t ysize() const { return size.ysize(); }
+  size_t oriented_xsize(bool keep_orientation) const {
```

Comment:
not a must, but if you want this new function could also be used in GetBasicInfo